geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -139,6 +164,33 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
     }
   }
 
+  public EventLoopGroup getSubscriberGroup() {
+    return subscriberGroup;
+  }
+
+  public synchronized void changeChannelEventLoopGroup(EventLoopGroup newGroup,
```

Comment:
I'd like to discuss what this should synchronize on. The method syncs on this (ExecutionHandlerContext) but the future it creates syncs on the channel. I think it would be better to sync on "channel" on this method instead of "this"

Reply:
Switched method synchronization to synchronize on `channel`